mono_repo/lib/src/yaml.dart

```diff
@@ -178,6 +201,15 @@ void _writeYaml(
         buffer.write(' ');
       }
       buffer.write('[]');
+    } else if (source.length == 1 && _isSimple(source.single)) {
```

Comment:
I am not sure this is really worth maintaining or even desirable? I personally wouldn't do this in hand written code in the first place (I think its better to have consistent style for yaml, and I much prefer not using any json inherited syntax).

Reply:
Understood. I think it's kinda nice to make more "conventional" output. But I can nix this one.

Reply:
Reverted these bits. PTAL